tutorials/static-site-generator.md

```diff
@@ -114,7 +114,7 @@ We'll parse these strings in our `load_content_items` function. Give the functio
 def load_content_items(content_strings):
     items = []
     for item in content_strings:
-        frontmatter, content = re.split("^\+\+\+\+\+$", item, 1, re.MULTILINE)
+        frontmatter, content = re.split("^\s*\+\+\+\+\+\s*$", item, 1, re.MULTILINE)
```

Comment:
added the whitespace character match to match even if the content_string is indented otherwise it will through an error if the line with the +++++ in the string is indented as it will not match, returning only one value (throwing `ValueError: not enough values to unpack (expected 2, got 1)`) since the regex is anchored to the start and end of line